### PR TITLE
PT-BR locale

### PR DESCRIPTION
**Pull request by glha**
I don't know if the add-on will accept PT-BR latin special chars. They are all UTF-8, but I can change them to html entities/codes, hex or even unicode (Ç = &Ccedil; = &#199; = &#xc7; = U+000C7). Please just let me know if that's the case and I'll promptly change them.
Cheers

### CODE REVIEW

**Comment by vaxul** [comment]
@glha

First of all, thank you for your contribution.

There is on major issue in your commit. You override the original EN texts in src/_locales/en/messages.json.

You have to copy this content into a new file src/_locales/pt/messages.json

And restore the original EN texts. 😉

**Comment by glha** [comment]
> There is one major issue in your commit. You override the original EN texts in src/_locales/en/messages.json.

Oh, sorry about that. That was quite nasty from me.

I've sent two new files to their correct paths, pt_BR and pt (main pt branch), this last one being a copy from the first. It may need some fine adjustments, like en_US and en_UK would need, but they are totally understandable between their speakers.

File: src/_locales/pt_BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{
+    "pluralRule": {
+        "message": "1"
```
Comment (vaxul) [inline src/_locales/pt_BR/messages.json]: Thank you for the update. Are currently `pt` and `pt_BR` identical? 

One main difference I might found is, that Brazilian Portuguese might need `pluralRule = 2` [as mentions here](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals#Plural_rule_2_(2_forms)), while Portuguese [has Plural Rule 1](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals#Plural_rule_1_(2_forms)).

We have [both rules implemented](https://github.com/projectdelphai/panorama-tab-groups/blob/master/CONTRIBUTION.md#add-language).

Comment (glha) [reply src/_locales/pt_BR/messages.json]: Yes, it should be rule 2. I tried to read some articles about plural rules, to understand better and don't mess the translation. By our definition, Zero has no means of plurality, so it should be singular.

```suggestion
        "message": "2"
```

**Pull request merged**